src/java/org/apache/cassandra/index/sai/disk/v1/bitpack/AbstractBlockPackedReader.java

```diff
@@ -34,6 +34,9 @@ public abstract class AbstractBlockPackedReader implements LongArray
     private final byte[] blockBitsPerValue;
     private final SeekingRandomAccessInput input;
 
+    private long prevTokenValue = Long.MIN_VALUE;
```

Comment:
```suggestion
    private long prevValue = Long.MIN_VALUE;
```
?

Reply:
Or, `previousValue`. I've changed it to that in my mission to make this all more readable.